core/pom.xml

```diff
@@ -76,11 +76,22 @@
         </dependency>
 
         <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <version>${jackson.version}</version>
+        </dependency>
+        <dependency>
             <groupId>org.json</groupId>
             <artifactId>json</artifactId>
             <version>${json.org.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.vividsolutions</groupId>
+            <artifactId>jts</artifactId>
+            <version>1.13</version>
+        </dependency>
```

Comment:
Nope, can't do that at the moment, because com.bedatadriven:jackson-datatype-jts:2.4 uses com.vividsolutions:jts:1.13 and I don't want to mess around with manual exclusion etc.

Can we put this on the to-be-resolved-before-next-release list?